native/spark-expr/src/utils.rs

```diff
@@ -72,9 +72,6 @@ pub fn array_with_timezone(
                 Some(DataType::Timestamp(_, Some(_))) => {
                     timestamp_ntz_to_timestamp(array, timezone.as_str(), Some(timezone.as_str()))
                 }
-                Some(DataType::Timestamp(_, None)) => {
```

Comment:
iirc this was introduced to fix cast issues caught by the fuzz tests which are not part of the ci. Can we run the fuzz tests again to verify that there are no regressions. 
Ideally, parquet_support should _not_ depend on any functions in utils.rs.

Reply:
+1 for removing this as I see that this came in as a part of https://github.com/apache/datafusion-comet/pull/1318